Modules/Devices/test/Devices.Application.Tests/TestDataGenerator.cs

```diff
@@ -35,16 +35,14 @@ public static Identity CreateIdentityWithOneDevice()
         return identity;
     }
 
-    public static Identity CreateIdentityWithApprovedDeletionProcess(DateTime approvalDate)
+    public static Identity CreateIdentityWithApprovedDeletionProcess(DateTime? approvalDate = null)
     {
-        var currentDateTime = SystemTime.UtcNow;
+        var dateTime = approvalDate ?? SystemTime.UtcNow;
```

Comment:
Just use 

```
approvalDate ??= SystemTime.UtcNow;
```
just like in all other `Create*` methods 😞

Reply:
The solution that you are suggesting was my initial approach, but then in line 43 `SystemTime.Set(approvalDate)` says that it cannot convert from 'System.Date.Time?' to 'System.DateTime' becase `approvalDate` is nullable. So putting it in a new variable that's not nullable `var dateTime = approvalDate ?? SystemTime.UtcNow` seemed like a good solution.

Reply:
```c# 
SystemTime.Set(approvalDate.Value);
```